samples/automl/automlTranslationDataset.js

```diff
@@ -0,0 +1,362 @@
+
+  // Get the full path of the dataset.
+  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+
+  // Get the multiple Google Cloud Storage URIs.
+  const inputUris = path.split(',');
+  const inputConfig = {
+    gcsSource: {
+      inputUris: inputUris,
+    },
+  };
+
+  // Import data from the input URI.
+  client
+    .importData({name: datasetFullId, inputConfig: inputConfig})
+    .then(responses => {
+      const operation = responses[0];
+      console.log('Processing import...');
+      return operation.promise();
+    })
+    .then(responses => {
+      // The final result of the operation.
+      if (responses[2].done === true) console.log(`Data imported.`);
+      console.log('Dataset imported.');
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_translation_importData]
+}
+
+function deleteDataset(projectId, computeRegion, datasetId) {
+  // [START automl_translation_deleteDataset]
+  const automl = require('@google-cloud/automl');
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetId = `Id of the dataset`;
+
+  // Get the full path of the dataset.
+  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+
+  // Delete a dataset.
+  client
+    .deleteDataset({name: datasetFullId})
+    .then(responses => {
+      const operation = responses[0];
+      return operation.promise();
+    })
+    .then(responses => {
+      // The final result of the operation.
+      if (responses[2].done === true) console.log('Dataset deleted.');
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_translation_deleteDataset]
+}
+
+require(`yargs`)
+  .demand(1)
+  .options({
+    computeRegion: {
+      alias: `c`,
+      type: `string`,
+      default: process.env.REGION_NAME,
+      requiresArg: true,
+      description: `region name e.g. "us-central1"`,
+    },
+    datasetName: {
+      alias: `n`,
+      type: `string`,
+      default: `testDataSet`,
+      requiresArg: true,
+      description: `Name of the Dataset`,
+    },
+    datasetId: {
+      alias: `i`,
+      type: `string`,
+      requiresArg: true,
+      description: `Id of the dataset`,
+    },
+    filter_: {
+      alias: `f`,
+      default: `translationDatasetMetadata:*`,
+      type: `string`,
+      requiresArg: true,
+      description: `Name of the Dataset to search for`,
+    },
+    multilabel: {
+      alias: `m`,
+      type: `string`,
+      default: false,
+      requiresArg: true,
+      description:
+        `Type of the classification problem, ` +
+        `False - MULTICLASS, True - MULTILABEL.`,

```

Comment:
Über-nit: should we use a string or enum here instead of a boolean? You can implement this using [Yargs' `.conflicts` function](https://github.com/yargs/yargs/blob/master/docs/api.md).